node/sync/src/block_sync.rs

```diff
@@ -319,6 +319,11 @@ impl<N: Network> BlockSync<N> {
             }
             // Update the latest height.
             current_height = self.canon.latest_block_height();
+            // Bump the timestamps of all outstanding requests if we successfully handled a BlockResponse.
+            let mut timestamps = self.request_timestamps.write();
+            for (_height, instant) in timestamps.iter_mut() {
+                *instant = Instant::now();
```

Comment:
```suggestion
            let new_timestamp = Instant::now();
            let mut timestamps = self.request_timestamps.write();
            for (_height, instant) in timestamps.iter_mut() {
                *instant = new_timestamp;
```
for a small perf boost